frontend/src/index.scss

```diff
@@ -108,3 +108,7 @@ $standardMargin: 10px;
   border: 1px solid rgb(198, 197, 214);
   border-radius: 4px;
 }
+
+.ag-rich-select-list {
+  height: auto !important;
+}
```

Comment:
Side quest: autosized the height of the dropdown selection to fit its content.

Before:
![before](https://github.com/mskcc/smile-dashboard/assets/86090707/d17ceacc-05f0-4f4a-9c63-9426ba0be8a9)

After:
![after](https://github.com/mskcc/smile-dashboard/assets/86090707/eb6221fc-21d8-4091-a8df-1367ec7ba167)

Reply:
thank you!